apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/account/Account.java

```diff
@@ -529,7 +529,7 @@ public long getEffectiveBalanceAPL(int height) {
         if (this.publicKey == null) {
             this.publicKey = getPublicKey(AccountTable.newKey(this));
         }
-        if (this.publicKey == null || this.publicKey.publicKey == null || height - this.publicKey.height <= 1440) {
+        if (this.publicKey == null || this.publicKey.publicKey == null || height - this.publicKey.getHeight() <= 1440) {
```

Comment:
Will we later use configurable constant instead of value ? ( <= **1440**)

Reply:
Such task is already planned: Refactor all nxt magic numbers to scalable config